pkg/cmd/run/list/list.go

```diff
@@ -122,16 +133,13 @@ func listRun(opts *ListOptions) error {
 		tp.AddField(run.Name, nil, nil)
 		tp.AddField(run.HeadBranch, nil, cs.Bold)
 		tp.AddField(string(run.Event), nil, nil)
+		tp.AddField(fmt.Sprintf("%d", run.ID), nil, cs.Cyan)
 
-		if opts.PlainOutput {
-			elapsed := run.UpdatedAt.Sub(run.CreatedAt)
-			if elapsed < 0 {
-				elapsed = 0
-			}
-			tp.AddField(elapsed.String(), nil, nil)
+		elapsed := run.UpdatedAt.Sub(run.CreatedAt)
+		if elapsed < 0 {
+			elapsed = 0
 		}
-
-		tp.AddField(fmt.Sprintf("%d", run.ID), nil, cs.Cyan)
+		tp.AddField(elapsed.String(), nil, nil)
```

Comment:
`elapsed` and `age` are two different things and both need their own column. the `AGE` column should be the difference between now and the run's `CreatedAt` timestamp. the `ELAPSED` column should be what is currently being calculated.